### PR TITLE
Update mad_strings.xml

### PR DESCRIPTION
**Pull request by LucaT69**
new stuff translated (italian 25 april) line from 938 to 995

### CODE REVIEW

**Comment by LucaT69** [comment]
latest translation (new stuff) and minor fix

File: packages/apps/Settings/res/values-it/mad_strings.xml [hunk 1]
```diff
@@ -935,5 +935,62 @@
...
+
+    <!-- Immersive recents -->
+    <string name="immersive_recents_title">Modalità recenti immersiva</string>
+    <string name="immersive_recents_default">Default</string>
+    <string name="immersive_recents_fullscreen">A tutto schermo</string>
+    <string name="immersive_recents_statusbar">Solo barra di stato</string>
+    <string name="immersive_recents_navbar">Solo barra di navigazione</string>
+
+    <!-- Recent clear all button and location -->
+    <string name="show_clear_all_recents_button_title">Rimuovi tutto (FAB)</string>
```
Comment (LucaT69) [inline packages/apps/Settings/res/values-it/mad_strings.xml]: Don't know what "FAB" is...
Sorry...

Comment (LucaT69) [reply packages/apps/Settings/res/values-it/mad_strings.xml]: Maybe FAB could stands for "FAB style" and AOSP stands for AOSP style ?

Comment (dragonpt) [reply packages/apps/Settings/res/values-it/mad_strings.xml]: FAB = Floating Action Button

its the "Clear All Button" but in a custom style

Currently on our rom, we only have "Clear All Button", this adds another Custom Button "FAB" mostly the same as the one we had on Lollipop RC5

Comment (LucaT69) [reply packages/apps/Settings/res/values-it/mad_strings.xml]: thank you!!!

**Pull request merged**